Fix backup codes column length for Argon2 values

ChristophWurst opened the pull request:
The new (php7.2) hasher's values are a bit longer than the ones of the old algorithm, hence we have to increase the column length.

- [x] Add a failing test that reproduces the issue
- [x] Fix the issue

Fixes #10766.

ChristophWurst committed:
Add an integration test that inserts long back codes values

Signed-off-by: Christoph Wurst <christoph@winzerhof-wurst.at>

ChristophWurst committed:
Fix backup codes column length to fix Argon2 hash values

Signed-off-by: Christoph Wurst <christoph@winzerhof-wurst.at>

ChristophWurst commented:
@rullzer I expected my first commit to trigger some CI errors, but it seems all okay. How's that possible? How can I find out if my tests are even executed?

rullzer commented:
@ChristophWurst good question let me have a look

rullzer commented:
Tests are not executed as per: https://github.com/nextcloud/server/blob/2c0743c46b8c338538a80e7e6bd01c33c59bacc6/tests/apps.php#L13

so that is why they don't fail :P

ChristophWurst committed:
Fix integration test execution on Dron

Apparently the Nextcloud server ignores tests that have `Integration` in their
fully-qualified class name, hence the backup codes integration tests were removed.
This moves them up one directory (out of `Integration`) to fix that.
Real unit tests remain in the `Unit` directory.

Signed-off-by: Christoph Wurst <christoph@winzerhof-wurst.at>

rullzer reviewed (approved):
:see_no_evil:

MorrisJobke committed:
raise version to trigger migration

Signed-off-by: Morris Jobke <hey@morrisjobke.de>

MorrisJobke reviewed (approved):
I added a version bump to trigger the migration 👍
